### ISSUE TITLE
Import Voices and VoiceSettings in generate() | No voice voice_id

### ISSUE THREAD

**Issue by RafaelGodoyEbert** [elevenlabs/elevenlabs-python#186]
Is there any way I can pull the voice from from elevenlabs import `voices` and still put `VoiceSettings` inside 
```
voice=Voice(
                     voice=all_voices ,
                     settings=VoiceSettings(stability=0.71, similarity_boost=0.5, style=0.0, use_speaker_boost=True)
                 ),
```

My code keeps giving error

```       
# This doesn't work
# Pull the voices from the voices which are names, Adam, Rachel, etc...


from elevenlabs import Voice, VoiceSettings, voices, generate

all_voices = voices()
audio = generate(
                 text=text_input_elevenlabs_api,
                 voice=Voice(
                     voice=all_voices , # Pull the voices from the voices which are names, Adam, Rachel, etc...
                     settings=VoiceSettings(stability=0.71, similarity_boost=0.5, style=0.0, use_speaker_boost=True)
                 ),
                 model="eleven_multilingual_v2",
                 api_key=id_api
             )
```

But if I put the indicated one, which is this

``` 
# Functional, but not practical

all_voices = voices()
from elevenlabs import Voice, VoiceSettings, voices, generate

audio = generate(
                 text=text_input_elevenlabs_api,
                 voice=Voice(
                     voice_id='EXAVITQu4vr4xnSDxMaL', # I wouldn't like to create a file and name it one by one, I wanted something practical like the voices but with an ID for here and in my interface there aren't several codes, I want the interface to return the names but the code identifies it as if it were the functional without settings=VoiceSettings
                     
                     settings=VoiceSettings(stability=0.71, similarity_boost=0.5, style=0.0, use_speaker_boost=True)
                 ),
                 model="eleven_multilingual_v2",
                 api_key=id_api
             )
```
Below Functional but without settings=VoiceSettings

```
# This is what I'm using, but I miss the possibility of settings=VoiceSettings
from elevenlabs import Voice, VoiceSettings, voices, generate

audio = generate(
                text=text_input_elevenlabs_api,
                voice=all_voices ,
                model="eleven_multilingual_v2",
                api_key=id_api
            )
```

**Comment by dsinghvi** [comment]
@RafaelGodoyEbert thanks for filing this error, we'll get back to you on how you can customize the setting for each voice and just identify with `name`.

**Comment by RafaelGodoyEbert** [comment]
But using `name`, the code does not identify, it returns an error.

```
Error generating audio: 1 validation error for Voice
voice_id
  field required (type=value_error.missing)
```

code updated according to message

```
            audio = generate(
                text=audio_input_elevenlabs_api,
                voice=Voice(
                    name=voice_model_input,
                    settings=VoiceSettings(stability=stability_elevenlabs, similarity_boost=similarity_boost_elevenlabs, style=style_elevenlabs, use_speaker_boost=use_speaker_boost_elevenlabs)
                ),
                model=modelos,
                api_key=id_api
            )
```

It's forcing me to have `voice_id=` inside voice to work
But if I put `voice_id=` it doesn't make sense to have `name=`

**Comment by dsinghvi** [comment]
@RafaelGodoyEbert This is being fixed on https://github.com/elevenlabs/elevenlabs-python/pull/196

**Comment by dsinghvi** [comment]
@RafaelGodoyEbert this is now fixed in `v1.0.0b0`. Please let us know if you have anymore issues!
